src/canmatrix/cmjson.py

```diff
@@ -199,32 +200,42 @@ def load(f, **options):
                 newframe.extended = 0
 
             for signal in frame["signals"]:
-                if "is_big_endian" in signal and signal["is_big_endian"]:
+                if signal.get("is_big_endian", False):
```

Comment:
```suggestion
                is_little_endian = signal.get("is_big_endian", False)
```

While we're refactoring...  :]  Do note that these are both a little different than before, though maybe more what we want, maybe.  In the left-side code the result was `True` if the value wasn't specified.  Not sure if you want to default to little or big endian.